homeassistant/components/google_assistant/helpers.py

```diff
@@ -397,6 +398,12 @@ def async_update(self):
         """Update the entity with latest info from Home Assistant."""
         self.state = self.hass.states.get(self.entity_id)
 
+        entity_config = self.config.entity_config.get(self.state.entity_id, {})
+
+        delay_time = entity_config.get(CONF_DELAY_TIME)
```

Comment:
Google integration can not depend on non HA standard attributes. It must be part of the normal alarm base component (to add stuff, and arch issue will need to be opened)

Reply:
fair point, to describe the scenario, unlike other google assistant traits, the alarm system expects the response to be the mode it set to, otherwise it reports an error ("something went wrong"), which is why I started adding async delays 'options', as most alarm system states aren't synchronous. Happy to have input on a more elegant way to do it.